### PR TITLE
add image embedding pipeline

### PR DESCRIPTION
**Pull request by reiase**
Signed-off-by: jie.hou <jie.hou@zilliz.com>

### CODE REVIEW

**Comment by oneseer** [comment]
the max num of models is three?

File: towhee/pipelines/image_embedding_pipeline.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        else:
+            models = model
+        num_branch = len(models)
+
+        models = dict(
+            zip([
+                'embedding_model_1',
+                'embedding_model_2',
+                'embedding_model_3',
+            ], models))
```
Comment (wxywb) [inline towhee/pipelines/image_embedding_pipeline.py:63]: it is implied that 3 embeddings ensemble is supported, if user want 4 or more, what does he need?

**Comment by wxywb** [comment]
Is normalization + aggregation + meta learner will use the same string 'ensemble' to configure?

**Comment by reiase** [comment]
> 

will be supported by operator ir

**Comment by reiase** [comment]
support short model name in image_embedding_api

**Pull request merged**